### ISSUE TITLE
code review

### ISSUE THREAD

**Issue by xiangl18** [leonshen95/DetectUtilityPoles#3]
You did a really good job! 
Here are my suggestions, it would be wonderful if they could help you.
The readme file does not directly show me the data structure, it’s train: test = 9:1, maybe the readme should include the percent of test data and train data, also there is no specific number of images in your own datasets, maybe the percent of test and train data should be adjusted due to the training result.
Also, maybe it is better to get a validation set, in case that you would like to change other models or hyperparameters, such as learning rate or batch size. For I saw your batch size is 24, maybe you could try to change it so that you could find the best batch size for recognizing utility poles to get a better result.
The function you build you own data set should be added. Maybe there could be a python file for help the users to build their own data set, for example, the program should have the function of renaming the images that the users used, and it can also separate the train, test and validation set, for the nest move. And also, the data augmentation function should be added in case the users have little number of images.
As for the images, you could consider image preprocessing so that you could get a better result, for example, you mention that the utility poles are hard to find in snow weather, maybe it could be solved by image preprocessing.
About the model, you choose YOLO v2 as your main algorithm, and use the pre-train model in PascalVOC, I don’t know whether you want to make your program to be real-time detection or not, and I haven’t seen your training model and result, maybe after you get a better GPU and get the result, you could consider change other algorithms, and make comparison with your current YOLO, so that you could find the algorithm suits the target most.
At last, you should consider making a program which could integrate all your function, so that the users could simply run one program to get all the Google Street view, the detecting result of the utility pole in the picture and finally the distance among the poles in map. I saw you got the examplecapture.py file, but it has not been finished, so maybe you should keep working on that?
